src/components/BannerSlider.tsx

```diff
@@ -108,7 +124,7 @@ class BannerSlider extends React.PureComponent<BannerSliderPanelProps, BannerSli
           style={{
             position: 'absolute',
             bottom: 0,
-            top: bannerHeight - 20,
+            top: this.state.heightImage ? this.state.heightImage - 20  : bannerHeight - 20,
```

Comment:
I think we could simply write bottom: 20 and drop the top field here. It worked when I tried it yesterday on my other branch, at least.